cmd/podman/create_cli.go

```diff
@@ -121,20 +139,20 @@ func validateVolumeOpts(option string) error {
 	for _, opt := range options {
 		switch opt {
 		case "rw", "ro":
+			foundRWRO++
 			if foundRWRO > 1 {
 				return errors.Errorf("invalid options %q, can only specify 1 'rw' or 'ro' option", option)
 			}
-			foundRWRO++
 		case "z", "Z":
+			foundLabelChange++
```

Comment:
Is this correct?  Aren't you erroring out on 'Z' above?  Also 'Z' is not listed in the man page if it's a valid argument.

Reply:
This is just a common function to validate the options being passed. They are all listed under the --volumes in the man page.